src/modules/IscsiClientLib.rb

```diff
@@ -473,12 +473,20 @@ def ScanDiscovered(data)
       target = ""
       portal = ""
       iface = ""
+      dumped = true
       Builtins.y2milestone("Got data: %1", data)
 
       Builtins.foreach(data) do |row|
         row = Builtins.substring(row, Builtins.findfirstnotof(row, "\t "), 999)
         if Builtins.search(row, "Target:") != nil
+          if !dumped
```

Comment:
btw having in method documentation example of output whould be quite helpful to understand this method.